tests/web/rpc/v1/test_trace_item_attribute_values_v1.py

```diff
@@ -202,7 +203,34 @@ def test_empty_results(self) -> None:
                     seconds=int((BASE_TIME + timedelta(days=1)).timestamp())
                 ),
             ),
+            key=AttributeKey(name="tag1", type=AttributeKey.TYPE_STRING),
             value_substring_match="this_definitely_doesnt_exist_93710",
         )
         res = AttributeValuesRequest().execute(req)
         assert res.values == []
+
+
+@pytest.mark.clickhouse_db
+@pytest.mark.redis_db
```

Comment:
`redis_db` fixture seems unused. Can be removed.

Reply:
yes it is, it's how the runtime config works